src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -257,8 +257,6 @@
                       (if (and platform/desktop? (= "right" (.-button (.-nativeEvent arg))))
                         (open-chat-context-menu message)
                         (do
-                          (when (= content-type constants/content-type-sticker)
-                            (re-frame/dispatch [:stickers/open-sticker-pack (:pack content)]))
```

Comment:
but we want this to be fixed later, maybe comment it better not remove completely?

Reply:
You can still find it in history when browsing the commit and PR?
Also ideally I don't think this sticker specific logic should be in this file.